projects/step-frontend/src/lib/os-plugins/modules/script-editor/components/script-editor/script-editor.component.html

```diff
@@ -1,14 +1,40 @@
 <mat-card>
   <mat-card-header>
-    <mat-card-title> Script Editor - Keyword: {{keyword?.attributes?.['name']}} </mat-card-title>
-    <button type="button" mat-icon-button (click)="execute()" matTooltip="Execute this plan" matTooltipPosition="below">
-      <step-icon name="play"></step-icon>
-    </button>
+    <mat-card-title> Script Editor - Keyword: {{keyword?.attributes?.['name']}}</mat-card-title>
+    <section>
+      <step-btn-group *ngIf="'kw-write' | hasRight | async" class="with-borders mr2">
+        <button
+          type="button"
+          mat-stroked-button
+          (click)="save()"
+          matTooltip="Save this plan"
+          matTooltipPosition="below"
+        >
+          <step-icon name="save"></step-icon>
+        </button>
+        <button
+          [ngClass]="{ 'color-green': noChanges && isAfterSave, 'color-black': !noChanges }"
```

Comment:
When we create css classes the name should reflect the elements status instead of the classes content. That way we could for example say we want to use blue as the saved color without changing the class name. 

Also for adding multiple classes it could be clearer to do it like this: (ngClass is also ok if you prefer it, but in that case I would add line-breaks):
[class.status-no-changes]=noChanges && !isAfterSave,
[class.status-has-changes]=!noChanges,
[class.status-saved]=noChanges && isAfterSave,

Reply:
Done